openshift-kube-apiserver/admission/customresourcevalidation/apiserver/validate_apiserver.go

```diff
@@ -246,14 +255,8 @@ func haveRequiredHTTP2CipherSuites(suites []string) bool {
 
 func validateMinTLSVersion(fieldPath *field.Path, version configv1.TLSProtocolVersion) field.ErrorList {
 	errs := field.ErrorList{}
-
-	if version == configv1.VersionTLS13 {
```

Comment:
do we know the reason why this check was added before? 

(I think this method is only called for a custom profile)

Reply:
Correct, this would have been a workaround to enable TLS1.3 is we did not block it.